nav2_recoveries/plugins/back_up.cpp

```diff
@@ -40,6 +40,7 @@ void BackUp::onConfigure()
     node_,
     "simulate_ahead_time", rclcpp::ParameterValue(2.0));
   node_->get_parameter("simulate_ahead_time", simulate_ahead_time_);
+  node_->get_parameter("transform_tolerance", transform_tolerance_);
```

Comment:
See above, these get params shouldn't happen, it should be in the backend recovery.hpp file. Look where this class is deriving from.